package.json

```diff
@@ -38,25 +38,25 @@
     "rollup.prod.ci": "rollup --config --config-prod --config-ci",
     "rollup.watch": "rollup --watch --config",
     "start": "npm run watch",
-    "test": "jest --coverage",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest --coverage",
```

Comment:
One thing that changes with prettier v3 is that now although they still have a commonjs entrypoint it actually does an `import` of a `.mjs` file, so the VM in which prettier runs now needs to have `.mjs` support. So to get it to run in Jest we have to pass this little argument here.

It all seems to work fine for now at least.